response.go

```diff
@@ -283,77 +285,15 @@ func visitModelNode(model interface{}, included *map[string]*Node,
 				node.ClientID = clientID
 			}
 		} else if annotation == annotationAttribute {
-			var omitEmpty, iso8601, rfc3339 bool
-
-			if len(args) > 2 {
-				for _, arg := range args[2:] {
-					switch arg {
-					case annotationOmitEmpty:
-						omitEmpty = true
-					case annotationISO8601:
-						iso8601 = true
-					case annotationRFC3339:
-						rfc3339 = true
-					}
-				}
-			}
 
 			if node.Attributes == nil {
 				node.Attributes = make(map[string]interface{})
 			}
```

Comment:
You moved this in to `marshalAttribute()` as well; is it still needed here?

Reply:
Ah right, no its not! Going to keep this one here and then delete the one in `marshalAttribute`. doing this here makes more sense.